tests/resources/Vsphere-UI-Util.robot

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  This resource contains any keywords dealing with web based operations being performed on a Vsphere instance
+
+*** Keywords ***
+Login To Vsphere UI
+    [Arguments]  ${url}=%{TEST_URL}  ${username}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}
+    Go To  ${url}
+    Wait Until Element Is Visible  username
+    Wait Until Element Is Enabled  username
+    Wait Until Element Is Visible  password
+    Wait Until Element Is Enabled  password
+
+    Input Text  username  ${username}
+    Input Text  password  ${password}
+
+    Wait Until Element Is Visible  submit
+    Wait Until Element Is Enabled  submit
+
+    Click Button  submit
+
+    Wait Until Page Contains  Summary
+    Wait Until Page Contains  Monitor
+    Wait Until Page Contains  Configure
+    Wait Until Page Contains  Permissions
+
+Navigate To VCH Creation Wizard
```

Comment:
Its better to navigate by URL if navigating through elements is not the scope of the test, which reduces some flakiness. We can have separate tests to verify navigate by elements specifically.

Reply:
I would rather not, I want to avoid direct URL navigation when at all possible

Reply:
But why? Those UI navigation steps will be repeated in every steps and could make our suite flaky.

Reply:
Because that is not how users use a product like this. Direct URL linking will simply mask problems that we will have later in the life cycle of the product.  We already have API tests that direct access endpoints, these tests are intentionally customer centric.